internal/dates.go

```diff
@@ -3,10 +3,14 @@ package internal
 import "time"
 
 var (
-	oneHourBeforeTGE   = time.Date(2023, time.October, 31, 13, 0, 0, 0, time.UTC)
-	TGE                = time.Date(2023, time.October, 31, 14, 0, 0, 0, time.UTC)
-	oneDayAfterTGE     = TGE.AddDate(0, 0, 1)
-	oneYearAfterTGE    = TGE.AddDate(1, 0, 0)
+	oneHourBeforeTGE           = time.Date(2023, time.October, 31, 13, 0, 0, 0, time.UTC)
+	TGE                        = time.Date(2023, time.October, 31, 14, 0, 0, 0, time.UTC)
+	oneDayAfterTGE             = TGE.AddDate(0, 0, 1)
+	oneYearAfterTGEMinusOneDay = TGE.AddDate(0, 0, 364) // October 29, 2024
+	oneYearAfterTGE            = TGE.AddDate(0, 0, 365) // October 30, 2024
```

Comment:
this is the fix. It looks like vesting accounts don't account for leap years. 2024 has 366 days because it is a leap year.

``go
// before
oneYearAfterTGE    = TGE.AddDate(1, 0, 0) // October 31, 2024

// after
oneYearAfterTGE            = TGE.AddDate(0, 0, 365) // October 30, 2024
```